packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -329,9 +329,8 @@ export class CeramicCliUtils {
         }
 
         const seed = u8a.fromString(cliConfig.seed)
```

Comment:
I guess we need to update the `seed` generation as well, right?

```js
if (!cliConfig.seed) {
  cliConfig.seed = u8a.toString(randomBytes(32))
  console.log('Identity wallet seed generated')
  await CeramicCliUtils._saveCliConfig(cliConfig)
}
```

Reply:
if it's 32 bytes it should be correct @simonovic86

Reply:
Everywhere in the tests we had 64 bytes for some reason 🤷

Reply:
nice! 🙂